src/fides/api/api/v1/endpoints/privacy_experience_endpoints.py

```diff
@@ -114,15 +134,26 @@ def privacy_experience_list(
 ) -> AbstractPage[PrivacyExperience]:
     """
     Public endpoint that returns a list of PrivacyExperience records for individual regions with
-    relevant privacy notices embedded in the response.
+    relevant privacy notices or tcf contents embedded in the response.
 
     'show_disabled' query params are passed along to further filter
     notices as well.
 
-    'fides_user_device_id' query param will stash the current preferences of the given user
-    alongside each notice where applicable.
+    :param db:
+    :param params:
+    :param show_disabled: If False, returns only enabled Experiences and Notices
+    :param region: Return the Experiences for the given region
+    :param component: Returns Experiences of the given component type
+    :param has_notices: Return if the Experience has content
+    :param has_config: If True, returns Experiences with copy. If False, returns just Experiences without copy.
+    :param fides_user_device_id: Supplement the response with current saved preferences of the given user
+    :param systems_applicable: Only return embedded Notices associated with systems.
+    :param request:
+    :param response:
+    :return:
     """
     logger.info("Finding all Privacy Experiences with pagination params '{}'", params)
+    content_required: bool = bool(has_notices)  # Renaming confusing query param
```

Comment:
As a first step, I've added an alias, which makes it read more clearly on the backend.

I don't think this is worth changing unless the level of effort is minimal. @allisonking what would be the effort required to update the frontend to use a new query param called `content_required` instead of `has_notices`?

Reply:
it's pretty minimal! we'd change it in two places:

fides-js
https://github.com/ethyca/fides/blob/094325a0f7b66c7f8d4207384ba3a60e55920883/clients/fides-js/src/services/fides/api.ts#L39

privacy center
https://github.com/ethyca/fides/blob/094325a0f7b66c7f8d4207384ba3a60e55920883/clients/privacy-center/features/consent/consent.slice.ts#L70

I can do it as part of `aking/3809/tcf-modal` (the main FE feature branch) if you'd like!

Reply:
Thanks for looking into this Allison - at this point I say we merge this, and can add a small follow-up this sprint